igvm/settings.py

```diff
@@ -162,6 +170,19 @@
     {'hypervisor': HYPERVISOR_ATTRIBUTES},
 ]
 
+AWS_DATACENTERS = [
+    'aws-fra',
+]
+
+AWS_RETURN_CODES = {
+    'pending': '0',
+    'running': '16',
+    'shutting-down': '32',
+    'terminated': '48',
+    'stopping': '64',
+    'stopped': '80',
```

Comment:
Shouldn't those be integers? Those are bit flags after all.

Reply:
Unfortunately the API is assuming these to be strings
```
botocore.exceptions.ParamValidationError: Parameter validation failed:
Invalid type for parameter Filters[0].Values[0], value: 0, type: <class 'int'>, valid types: <class 'str'>
Invalid type for parameter Filters[0].Values[1], value: 16, type: <class 'int'>, valid types: <class 'str'>
Invalid type for parameter Filters[0].Values[2], value: 32, type: <class 'int'>, valid types: <class 'str'>
Invalid type for parameter Filters[0].Values[3], value: 48, type: <class 'int'>, valid types: <class 'str'>
Invalid type for parameter Filters[0].Values[4], value: 64, type: <class 'int'>, valid types: <class 'str'>
Invalid type for parameter Filters[0].Values[5], value: 80, type: <class 'int'>, valid types: <class 'str'>
```

Reply:
I made them Integers and only cast them to String when calling `aws_describe_instance_status`. This makes also return code comparison a lot easier